common/app/layout/Front.scala

```diff
@@ -438,7 +439,8 @@ object Front extends implicits.Collections {
     pressedPage.collections.filterNot(_.curatedPlusBackfillDeduplicated.isEmpty).zipWithIndex.mapAccumL(
         (Set.empty[TrailUrl], initialContext, emptyDedupedResultWithPath)
       ) { case ((seenTrails, context, dedupedFrontResult), (pressedCollection, index)) =>
-        val container = Container.fromPressedCollection(pressedCollection)
+        val omitMPU = pressedPage.omitMPUsFromContainers(edition)
```

Comment:
It comes from the `AdSlotAgent`, which reads a json doc every minute that has the takeover details.